modes/magit-todos/evil-collection-magit-todos.el

```diff
@@ -39,6 +39,11 @@
   (evil-collection-define-key 'normal 'magit-status-mode-map
     "gT" (and (bound-and-true-p magit-todos-mode) 'magit-todos-jump-to-todos)))
 
+(defun evil-collection-magit-todos-supress-warning (func &rest args)
+  "Supress the jT keybinding warning."
+  (cl-letf* (((symbol-function 'message)
+              (lambda (&rest _) nil)))
```

Comment:
Couldn't we replace the lambda by `#'ignore` and `(symbol-function 'message)` by `#'message`?

Reply:
It seems that `(symbol-function 'message)` is inreplaceable from [this answer](https://stackoverflow.com/questions/39550578/in-emacs-what-is-the-difference-between-cl-flet-and-cl-letf)